dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageProblem.java

```diff
@@ -150,9 +150,21 @@ public String formatSymbolProblem() {
   public static String formatLinkageProblems(Set<LinkageProblem> linkageProblems) {
     StringBuilder output = new StringBuilder();
 
+    // Don't group AbstractMethodProblems by symbols because they do not fit in the
+    // "... referenced by ..." format.
+    ImmutableSet.Builder<AbstractMethodProblem> abstractMethodProblems = ImmutableSet.builder();
+    ImmutableSet.Builder<LinkageProblem> problemsToGroupBySymbols = ImmutableSet.builder();
+    for (LinkageProblem linkageProblem : linkageProblems) {
+      if (linkageProblem instanceof AbstractMethodProblem) {
```

Comment:
There's probably a cleaner way to do this without instanceof.

Reply:
It might require some reorganizing of how and where we group things. This is OK for now. I'll take a look at this when I get a minute.

Reply:
Thank you. I created https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1618 as a followup.